cmd/dagger/call.go

```diff
@@ -16,7 +16,7 @@ var outputPath string
 var jsonOutput bool
 
 var callCmd = &FuncCommand{
-	Name:  "call [flags] [FUNCTION]...",
+	Name:  "call [flags] [function]...",
```

Comment:
This will produce:
```
dagger call [flags] [function]...
dagger call [flags] [command]
```

Which isn't right because here, "function" and "command" are the same thing. As per the conventions:

> When multiple synopsis lines are given for a utility, it is an indication that the utility has mutually-exclusive arguments.

This happens because you're describing _function_ as a positional argument, when it's actually a subcommand. In the template, cobra sees that it has subcommands so it adds the second line.

Docker, for example, doesn't print two lines. It always prints the same if it has subcommands:

```
{{- if not .HasSubCommands}}  {{.UseLine}}{{end}}
{{- if .HasSubCommands}}  {{ .CommandPath}}{{- if .HasAvailableFlags}} [OPTIONS]{{end}} COMMAND{{end}}
```

In our case, we'd have to check if the command is required or not. That's a plus in docker for consistency that we don't have.

In this case the subcommand is indeed optional, but you mark that by making the usage about the `call` command itself and not include children. If you change the usage to just `call` and remove the square brackets in `[command]` in the template, it produces this which is more correct:

```
dagger call [flags]
dagger call <command> [flags]
```

But Cobra's default is actually `[command]` and in the cases where the command is actually required, then the first line doesn't make sense.

Since we have a mix (sometims required, sometimes optional), we may have to use a helper function to make a single usage line instead, that detects which applies. But I suggest keeping it for another PR and simply keep the usage here as:

```suggestion
	Name:  "call",
```

Reply:
Now that `[flags]` was renamed to `[options]`, we need to add it explicitly. As for the two usage lines in the template, I don't think we can generalize because we have a command that requires a subcommand, and another where it's optional. Better be explicit in `cmd.Use`. I pushed an update with these changes so I can merge this quickly.